src/status_im2/common/contacts/events_test.cljs

```diff
@@ -0,0 +1,59 @@
+(ns status-im2.common.contacts.events-test
+  (:require [cljs.test :refer-macros [deftest is run-tests]]
+            [status-im2.common.contacts.events :as core]))
+
+;; initial db
+(def init-db
+  {:networks/current-network "mainnet_rpc"
+   :networks/networks        {"mainnet_rpc"
+                              {:id     "mainnet_rpc"
+                               :config {:NetworkId 1}}}})
+
+;; new identity (:searching state)
```

Comment:
I don't understand the need for this top-level comment.

Reply:
comment removed